bindings/resolver.go

```diff
@@ -0,0 +1,253 @@
+package bindings
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+// SecretReader represents the read-only content of a secret value.
+type SecretReader struct {
```

Comment:
I suppose I chose it because it felt most descriptive of its primary purpose. I feel you though and am not super attached to the name.

Given the other thread about ranaming `Binding.Secret` to `Binding.Entries`, perhaps `Entry` would make sense name here?

Reply:
Done.